### PR TITLE
Update deployment docs to include China Solution

### PR DESCRIPTION
**Pull request by Moto-Venturer**
I noticed many of the "Other Services" listed on this page do not provide service in China.

 I have updated the lists to include 21YunBox's Next.js China deployment guide/starter to the Other Services 'Managed Servers' and 'Static only' lists.

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `pnpm lint`
- [ ] The examples guidelines are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: docs/deployment.md [hunk 1]
```diff
@@ -113,6 +114,8 @@ The following services support deploying Next.js using [`next export`](/docs/adv
 - [Cloudflare Pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-nextjs-site/)
 - [Firebase](https://github.com/vercel/next.js/tree/canary/examples/with-firebase-hosting)
 - [GitHub Pages](https://github.com/vercel/next.js/tree/canary/examples/github-pages)
+- [21YunBox](https://www.21cloudbox.com/solutions/how-to-deploy-nextjs-project-in-production-server.html)
+
```
Comment (leerob) [inline docs/deployment.md:118]:
```suggestion
```

Comment (leerob) [reply docs/deployment.md:118]: If it works as a server, then it probably shouldn't be listed under static only.

File: docs/deployment.md [hunk 2]
```diff
@@ -102,6 +102,7 @@ The following services support Next.js `v12+`. Below, you’ll find examples or
 - [Heroku](https://elements.heroku.com/buildpacks/mars/heroku-nextjs)
 - [Railway](https://railway.app/new/starters/nextjs-prisma)
 - [Render](https://render.com/docs/deploy-nextjs-app)
+- [21YunBox](https://www.21cloudbox.com/solutions/how-to-deploy-nextjs-project-in-production-server.html)
```
Comment (leerob) [inline docs/deployment.md:105]: This article is from 2020 - is it still up to date?

Comment (Moto-Venturer) [reply docs/deployment.md:105]: Hi @leerob,

Good question.
Yes, I've confirmed the article content is up to date, and still works to deploy Next.js into China.

**Comment by leerob** [comment]
Thanks for suggesting this change – I want to make sure before we introduce a service into the hosting section, we've seen multiple folks successfully deploy Next.js applications there and there are ample resources for them to succeed. 

For example, DigitalOcean has done a really great job: https://docs.digitalocean.com/tutorials/app-nextjs-deploy/

Let's move this back to the issue stage and we can revisit this PR. Thank you.

**Pull request closed without merging**